docs/api.md

```diff
@@ -53,12 +53,13 @@ If `opts` is specified, then the default options (shown below) will be overridde
 
 ```js
 {
-  maxConns: Number,        // Max number of connections per torrent (default=55)
-  nodeId: String|Buffer,   // DHT protocol node ID (default=randomly generated)
-  peerId: String|Buffer,   // Wire protocol peer ID (default=randomly generated)
-  tracker: Boolean|Object, // Enable trackers (default=true), or options object for Tracker
-  dht: Boolean|Object,     // Enable DHT (default=true), or options object for DHT
-  webSeeds: Boolean        // Enable BEP19 web seeds (default=true)
+  maxConns: Number,         // Max number of connections per torrent (default=55)
+  nodeId: String|Buffer,    // DHT protocol node ID (default=randomly generated)
+  peerId: String|Buffer,    // Wire protocol peer ID (default=randomly generated)
+  tracker: Boolean|Object,  // Enable trackers (default=true), or options object for Tracker
+  dht: Boolean|Object,      // Enable DHT (default=true), or options object for DHT
+  dhtState: Boolean|String, // Persist DHT nodes (default=true), or save file path
```

Comment:
I think this would be clearer as two separate variables. A boolean for whether to persist the state, and a file path for persisting the state

Maybe named (doesn't matter)
``` js
{
  persistDht: Boolean,
  persistDhtPath: String
}
```

Reply:
I've updated to split them that way.